AnonUpload/src/main/java/com/theironyard/controllers/AnonFileController.java

```diff
@@ -45,4 +64,19 @@ public void upload(MultipartFile file, HttpServletResponse response) throws IOEx
     public List<AnonFile> getFiles(){
         return repository.findAll();
     }
+
+    @RequestMapping(path = "/delete",method = RequestMethod.POST)
+    public void deleteFile(String fileName, String password, HttpServletResponse respone) throws Exception {
+        AnonFile fileToDelete = repository.findByFilename(fileName);
+        if(fileToDelete.getPassword().isEmpty()){
```

Comment:
This can throw a `NullPointerError` if password is null especially since the column is nullable.